tests/integration/flask/conftest.py

```diff
@@ -47,7 +47,7 @@ def fixture_test_db_flask_image(pytestconfig: Config):
 @pytest_asyncio.fixture(scope="module", name="charm_file")
 async def charm_file_fixture(pytestconfig: pytest.Config, ops_test: OpsTest) -> pathlib.Path:
     """Get the existing charm file."""
-    charm_file = pytestconfig.getoption("--charm-file")
+    charm_file = next(f for f in pytestconfig.getoption("--charm-file") if "flask-k8s" in f)
```

Comment:
Same here

Reply:
Same issue as in previous comment. Also fixed.